src/Util.jl

```diff
@@ -0,0 +1,55 @@
+import StaticArrays
+import StaticArrays: SVector, MVector
+import CUDA
```

Comment:
We can make this optional and added only when needed since we also depend on AMDGPU.jl, see [here](https://github.com/JuliaORNL/GrayScott.jl/blob/GrayScott-JACC/src/simulation/Simulation.jl#L20)